File: Gombang/Weekly Contest 397/3146_Permutation_Difference_between_Two_Strings.cs
public class Solution
{
    // 단순하게 이중 포문을 이용하여 인덱스의 차이 값을 계산하여 풀이하였습니다.
    public int FindPermutationDifference(string s, string t)
    {
        int sum = 0;

        for (int i = 0; i < s.Length; i++)
        {
            for (int j = 0; j < t.Length; j++)
            {
                if (s[i] == t[j])
                {
                    sum += Math.Abs(i - j);
                    break;
                }
            }
        }

        return sum;
    }   
}

File: Seori/Weekly Contest 397/3146. Permutation Difference between Two Strings.py
class Solution:
    def findPermutationDifference(self, s: str, t: str) -> int:
        dict_index = {}

        # [1] Create a dictionary with the key as the character and the value as a list of indices
        for i in range(len(s)):
            dict_index[s[i]] = [i]
        
        # [2] Append the index of the character to the list
        for i in range(len(t)):
            dict_index[t[i]].append(i)
        
        # [3] Return the sum of the absolute difference between the indices
        answer = 0
        for a, b in dict_index.values():
            answer += abs(a-b)

        return answer